model/patch/intent.go

```diff
@@ -0,0 +1,56 @@
+package patch
+
+import (
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/pkg/errors"
+	"gopkg.in/mgo.v2/bson"
+)
+
+// Intent represents an intent to create a patch build and is processed by an amboy queue.
+type Intent interface {
+	// ID returns an identifier such that the tuple
+	// (intent type, ID()) is unique in the collection.
```

Comment:
Is this comment still accurate?

Reply:
Yes, though only because ID is unique to the collection.